src/app/pages/user-profile/UserProfile.css

```diff
@@ -82,6 +82,35 @@ body {
   appearance: none;
 }
 
+.button-edit {
+  text-align: center; /* Centra el contenido del botón */
```

Comment:
Delete comments please, unless it is important to clarify something relevant.

Reply:
Done.